src/plays/simple-live-chat/components/chat/chat-body.scss

```diff
@@ -0,0 +1,107 @@
+@import "../../variables";
+
+.chat-body {
```

Comment:
Lets use a unique class name for styles to prefix as we did with the previous plays.

Reply:
yes done